profiles/base.yaml

```diff
@@ -234,6 +234,8 @@ meditation_pause_timer: 20
 # You won't try to activate a barb buff until you have this much inner fire.
 # You need some inner fire in reserve to pay the periodic cost of abilities.
 barb_buffs_inner_fire_threshold: 15
+# Minimum inner fire to trigger analyze flame during whirlwinding
+barb_combat_inner_fire_threshold: 10
```

Comment:
The comment and the name of the var in `combat-trainer` both have 'flame' in the name, what do you think about including it here for the setting?